examples/ios-objc/README.md

```diff
@@ -0,0 +1,47 @@
+# RudderTyper - iOS (Objective C)
+
+This is a sample iOS (Objective C) app designed to demonstrate how to use the RudderStack iOS SDK to track the RudderTyper generated events.
+
+## Prerequisites
+
+- Download, install and setup [Xcode](https://apps.apple.com/us/app/xcode/id497799835?mt=12).
+
+## Getting Started
+
+### Step 1: Open project in Xcode
+
+- Open project path in Terminal & run `pod update`. Next open the `RudderTyperExample.xcworkspace`, located under `examples/ios-objc`, in Xcode.
+
+### Step 2: Configure RudderStack Properties
+
+- Update the `writeKey`, `dataPlaneUrl`, and `controlPlaneUrl` with your specific values in the `RudderConfig.plist`.
+
+  ![RudderConfig](screenshots/rudderConfig_path.png)
+
+### Step 3: Add/Regenerate Analytics Files
+
+- Run the following command to regenerate the client library based on your tracking plan using the production version of analytics files.
+
+```
+npx rudder-typer@latest init
+```
+
+- To regenerate the client library based on your tracking plan using the analytics files from the local repository, use the predefined shellscript file.
+
+```
+./rt_local_script.sh
```

Comment:
This shell script takes an argument as well, indicates the command to run.

Reply:
What will be the possible argument?

Reply:
All the supported commands of rudder-typer.